lib/ios/BackpackReactNative/Classes/CalendarBridge/RCTBPKCalendarManager.m

```diff
@@ -121,6 +122,21 @@ - (RCTBPKColorBucket *_Nullable)colorBucketForDate:(NSDate *)date
     return nil;
 }
 
+- (BOOL)calendar:(BPKCalendar *)calendar isDateEnabled:(NSDate *)date {
+    NSAssert([calendar isKindOfClass:RCTBPKCalendar.class],
+             @"calendar value is not of type RCTBPKCalendar as expected.");
+    if (![calendar isKindOfClass:RCTBPKCalendar.class]) {
+        return BPKCalendarDateCellStyleNormal;
+    }
+
+    RCTBPKCalendar *rctCalendar = (RCTBPKCalendar *)calendar;
+    if (rctCalendar.rct_disabledDates == nil) {
```

Comment:
This check is somewhat redundant because it also gets checked on line 137 due to Objective-C's `nil` handling

Reply:
I think it's more readable to be explicit